relay-event-schema/src/protocol/span.rs

```diff
@@ -280,6 +288,7 @@ mod tests {
             sentry_tags: ~,
             received: ~,
             measurements: ~,
+            _metrics_summary: ~,
```

Comment:
Let's add a `_metrics_summary` to the test input so we can see that it's properly transferred to the span.